src/oc/auth/api/google.clj

```diff
@@ -14,14 +15,14 @@
 
 (defn- redirect-to-web-ui
   "Send them back to a UI page with an access description ('google' or 'failed') and a JWToken."
-  ([redirect access]
+  ([redirect-origin redirect access]
     (redirect-to-web-ui redirect access nil :not-a-new-user)) ; nil = no jwtoken
```

Comment:
@cjsauer i think you are missing a parameter here, shouldn't you pass also redirect-origin in?

Reply:
@bago2k4 good catch, fixed.